src/components/Sidebar/Sidebar.less

```diff
@@ -169,6 +169,12 @@
 			& > .lucid-Sidebar {
 				flex: 1;
 			}
+
+			// Addresses a Firefox specific rendering issue with a Submarine nested in a Sidebar.
+			& > .lucid-Submarine, x:-moz-any-link, x:default { 
```

Comment:
That's a really good question... I got it from http://browserhacks.com/ as a method of selecting for Firefox specifically, and based on my testing, it's accomplishing that.

I'll do some reading to try and figure out what it actually means.